schema/Repos.js

```diff
@@ -32,10 +32,6 @@ cube(`Repos`, {
     type: {
       sql: `type`,
       type: `string`
-    },
-    avgPullAge:{
```

Comment:
I think this is needed though @harryli256

Reply:
Please re-review!